lib/layout/dummyLayoutV1.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * dummy-layout.js
+ *
+ * this dumps a fake graph
+ *
+ * @author John O'Brien.
+ */
+
+var Node = require('../model/node.js');
+var Graph = require('../model/graph.js');
+var Port = require('../model/port.js');
+
+// Create a graph instance.
+var graph;
+
+function buildGraph(nodes_and_connections) {
+	var nodes = [];
+	//loop for building the nodes
+	for(var i=0; i<4; i++){
+		//creates the ports
+		var portIn =new Port((i*100)+01, "input");
+		if(i==3){
+			portIn.setLocation("S", 0);
```

Comment:
I have some enums in my branch. If those become reality then we can use them here. Keep an eye out.